### ISSUE TITLE
ViroNode is not rotating ViroCamera

### ISSUE THREAD

**Issue by dominicrico** [viromedia/viro#304]
The following example is not working. The ViroNode is rotated but the ViroCamera as a child of the node is not rotated in the axis of the Node rotation.

```
<ViroNode rotation={[0, this.props.cameraRotation[1], 0]} position={[0, 0, 0]}>
  <ViroCamera position={[0, 0, 0]} rotation={[-this.props.cameraRotation[0], 0, 0]} active={true} />
</ViroNode>
```

I'm using a Viro3DSceneNavigator without AR/VR.

Thanks for your help.

**Comment by radvani** [comment]
Hi @dominicrico, 

Thanks for reporting this. We reproduced and fixed the bug, and the fix will be available in our next release.

Raj

**Comment by radvani** [comment]
This is now fixed in ViroReact 2.8.0. Thanks!
